src/HomeWork/Month.java

```diff
@@ -0,0 +1,135 @@
+package HomeWork;
+
+public class Month {
+    public static void longestWordOfThree() {
+        String str1 = "Alan";
```

Comment:
We should provide special values only as arguments during calling the method in the main method because a method should be universal but your method can work only with 3 specific words

 public static void longestWordOfThree(String word1, String word2, String word3)

Reply:
and call the method in the main method:
longestWordOfThree("Alan", "Alice", "Wake")